corehq/apps/dump_reload/management/commands/load_domain_data.py

```diff
@@ -47,13 +51,14 @@ def handle(self, dump_file_path, **options):
         total_object_count = 0
         model_counts = Counter()
         loaders = options.get('loaders')
+        object_filter = options.get('object_filter')
         if loaders:
             loaders = [loader for loader in LOADERS if loader.slug in loaders]
         else:
             loaders = LOADERS
 
         for loader in loaders:
-            loader_total_object_count, loader_model_counts = self._load_data(loader, extracted_dir)
+            loader_total_object_count, loader_model_counts = self._load_data(object_filter, loader, extracted_dir)
```

Comment:
`object_filter` is the third parameter of `_load_data()`, but this is passing the value in the first parameter.

Reply:
:+1: 014a6dc30ac